### PR TITLE
[435.433.482]  optimized

### PR DESCRIPTION
**Pull request by XuMinhui**

### CODE REVIEW

File: src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx [hunk 1]
```diff
@@ -51,11 +51,41 @@ export const StoriesSlider = () => {
...
+      ? liveDrops
+      : comingDrops.length !== 0
+      ? comingDrops
+      : prevDrops;
+  // debugger
 
+  const renderedItems = showDrop.map(item => {
+    const chips = (state: number) => {
+      const label =
+        state === SearchDropsParamState.Live
```
Comment (0123cf) [inline src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx]: use object map ?

ex
```
   {
[label]: conversionValue
}
```

File: src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx [hunk 2]
```diff
@@ -51,11 +51,41 @@ export const StoriesSlider = () => {
...
+    requestKey: DROPS_PREV_KEY,
+  });
+
   const liveDrops = data?.items || [];
+  const comingDrops = dataComing?.items || [];
+  const prevDrops = dataPrev?.items || [];
 
-  const renderedItems = liveDrops.map(item => {
-    const chips = <StoriesChip label="live" isLive />;
+  const showDrop =
```
Comment (0123cf) [inline src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx]: const showDrop = data?.items ?? dataComing?.items ?? dataPrev?.items ?? []

delete  liveDrops and comingDrops and prevDrops

File: src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx [hunk 3]
```diff
@@ -51,11 +51,41 @@ export const StoriesSlider = () => {
...
+    liveDrops.length !== 0
+      ? liveDrops
+      : comingDrops.length !== 0
+      ? comingDrops
+      : prevDrops;
+  // debugger
```
Comment (Pszz) [inline src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx]: Remove  it.
` // debugger `

File: src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx [hunk 4]
```diff
@@ -51,11 +51,41 @@ export const StoriesSlider = () => {
...
 
+  const renderedItems = showDrop.map(item => {
+    const chips = (state: number) => {
+      const label =
+        state === SearchDropsParamState.Live
+          ? 'Live'
+          : state === SearchDropsParamState.Coming
+          ? 'Coming'
+          : 'Previous';
```
Comment (Pszz) [inline src/modules/drops/screens/Drops/components/StoriesSlider/StoriesSlider.tsx]: 多语言？

**Review by 0123cf — APPROVED** [review]

**Review by Pszz — APPROVED** [review]

**Pull request merged**